test/framework/integration_framework/integration_test_framework.hpp

```diff
@@ -153,8 +153,12 @@ namespace integration_framework {
      * @param statuses - list of statuses to check against
      * @return this
      */
-    IntegrationTestFramework &sendTx(const shared_model::proto::Transaction &tx,
-                                     std::set<TxStatus> statuses);
+    IntegrationTestFramework &sendTx(
+        const shared_model::proto::Transaction &tx,
```

Comment:
Also, can we work with interfaces here?

Reply:
it's consistent with ITF interface, so what's the point?